Controllers/Mobile/v1/ProjectsController.cs

```diff
@@ -528,5 +528,31 @@ public async Task<IActionResult> GetProjectCommentsAsync([FromRoute] long projec
             return Ok(CreateSuccessResponse(paginatedCommentOutDTOs));
         }
 
+        [Authorize(Roles = Constants.USER_TYPE_CLIENT)]
+        [HttpPatch("{projectId}/completed")]
+        public async Task<IActionResult> MarkProjectAsCompletedAsync([FromRoute] long projectId)
+        {
+            if (!ModelState.IsValid)
+                return CustomBadRequest();
+
+            long authenticatedClientId = authService.GetUserId((ClaimsIdentity)HttpContext.User.Identity);
+            string nameOfAuthenticatedClient = authService.GetNameOfUser((ClaimsIdentity)HttpContext.User.Identity);
```

Comment:
I don't see the use of this variable, please explain it to me. Also, if you need the nameOfAuthenticatedClient you  can get the name of the client from the authenticatedUser variable you have defined below.